crates/rome_js_formatter/src/js/assignments/object_assignment_pattern_property.rs

```diff
@@ -21,7 +21,7 @@ impl FormatNodeFields<JsObjectAssignmentPatternProperty>
         formatted![
             formatter,
             [
-                member.format(),
+                FormatMemberName::from(member?).format(formatter),
```

Comment:
Nit: You can remove the `.format` here. 
```suggestion
                FormatMemberName::from(member?),
```